spec/rails_helper.rb

```diff
@@ -0,0 +1,74 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+
+ActiveRecord::Migration.maintain_test_schema!
+
+def login_user
+  visit "/"
+  click_link "Log In with Facebook"
+end
+
+def create_place
+  Place.create(name: "turing school of software design", place_id: 123)
+end
+
+def create_user
+  User.create(email: "Ryan@yeah.com",
+           uid: "123abc",
+           name: "Ryan Asensio",
+           oauth_token: "xxx11")
+end
+
+def stub_omniauth
+  OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+    provider: 'facebook',
+    uid: '123',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.